src/utils/custom_yaml.py

```diff
@@ -0,0 +1,10 @@
+import json
+import re
+import yaml
```

Comment:
This is a non-standard library and since most of script server users don't need it, the module should be optional.
In order to make it work for others, the import statement should be moved inside `loads` function. I.e. it will be imported only when we try to read a yaml file.